joomla/40-backend-template
# moduleorder.js refactor

Pull Request for Issue # .
### Summary of Changes
- Vanilla for module order field
- move static global functions out of core.js
### Testing Instructions

Go to Admin->edit module
Change the position of the module
Check xhr and the module oder element

Also in `administrator/components/com_modules/views/module/tmpl/edit.php`
after line 15 insert `JHtml::_('formbehavior.chosen', '.advancedSelect', null, array('disable_search_threshold' => 0));` and re-test (this will enable chosen)
### Documentation Changes Required
